### ISSUE TITLE
Speed, range and odometer in miles

### ISSUE THREAD

**Issue by kernalkue** [Dielee/volvo2mqtt#34]
**Is your feature request related to a problem? Please describe.**
I live in a silly country that uses miles instead of kilometres

**Describe the solution you'd like**
An option to specify the units as imperial would be fantastic (for speed, range and odometer)

**Describe alternatives you've considered**
I could create my own sensors to piggyback off yours.

**Additional context**
None

**Comment by Dielee** [comment]
This is already done. Just change your babel locale to `en_US`

**Comment by kernalkue** [comment]
Any chance it could work with en_GB?

**Comment by Dielee** [comment]
Why ? This is the locale for GB, they are using metric units...

**Comment by kernalkue** [comment]
GB (the UK) uses a stupid mix.
Distance (when travelling in a vehicle) is in miles
Speed (when travelling in a vehicle) is in miles/hour
Fuel (non-electric) is metric though, in litres
Temperature is also metric, in Celcius

I've also noted that my units have changed, but the values are still the metric values (i.e. they are too large), they need converting.

**Comment by Dielee** [comment]
Oh jesus... Does the app converts to the right units and distances ?

**Comment by kernalkue** [comment]
The "Volvo cars" app?
It doesn't have anywhere near the number of sensors available that you have.
With the driving journal on, it does show instances in miles and there is odometer in there which has the right value and units

**Comment by Dielee** [comment]
Okay... are you able to send me the conversion from metric to imperial for distances and speeds ?

**Comment by kernalkue** [comment]
I'm wondering if I can use your existing multipliers/dividers...

**Comment by kernalkue** [comment]
FYI 1km = 0.621371 miles

**Comment by Dielee** [comment]
Should work. Are you able to run a docker container outside HA OS to test the testbuilds ?

**Comment by kernalkue** [comment]
Yes probably, but not quickly. Happy to look later if that helps.

Using the odometerMultiplier I have got the odo value correct.
There isn't one I can use for "Range"

**Comment by Dielee** [comment]
Yes, they are not used to be for unit conversation... I will add a unit conversation for `en_GB`. 
This system is used all over in the UK ? Or only partly ?

**Comment by kernalkue** [comment]
All of the UK (England/Scotland/Wales/Northern Ireland)

**Comment by Dielee** [comment]
Okay, let me know when you are ready to test some builds ;)

**Comment by Dielee** [comment]
Please test if this build works for you: `ghcr.io/dielee/volvo2mqtt:sha-cc88907`

**Comment by kernalkue** [comment]
Ok! Tested it.

Looks good, apart from you should use "mph" not "mp/h".

I don't understand the average speed value. It's saying "1.86", but at the moment I don't know what it should read, so not sure what to suggest. Also the "Average Fuel Consumption" just always says "50.0 liters", which I assume is just a meaningless value for my BEV.

**Comment by Dielee** [comment]
Done with v1.6.0
